test/test_l2_projection.jl

```diff
@@ -267,13 +267,14 @@ function test_export(;subset::Bool)
     end
 
     mktempdir() do tmp
-        fname = vtk_grid(joinpath(tmp, "projected"), grid) do vtk
-            vtk_point_data(vtk, p, p_scalar, "p_scalar")
-            vtk_point_data(vtk, p, p_vec, "p_vec")
-            vtk_point_data(vtk, p, p_tens, "p_tens")
-            vtk_point_data(vtk, p, p_stens, "p_stens")
+        fname = joinpath(tmp, "projected")
```

Comment:
@fredrikekre: Note this change. Should it be kept that close(::VTKFile) returns a vector with a single element being the filename when the normal close does not?

Reply:
I don't think it is very useful to get the filenames